docs/source/quickstart.rst

```diff
@@ -158,38 +171,55 @@ Once your model is finetuned, use it for prediction anywhere you want!
     # load finetuned checkpoint
     model = ImageClassifier.load_from_checkpoint("image_classification_model.pt")
 
-    predictions = model.predict('path/to/your/own/image.png")
+    predictions = model.predict('path/to/your/own/image.png')
 
 ----
 
 Train
 -----
 When you have enough data, you're likely better off training from scratch instead of finetuning.
+Steps here are similar to finetune:
 
-.. code-block:: python
-
-    from torch import nn, optim
-    from torch.utils.data import DataLoader, random_split
-    from torchvision import transforms, datasets
-    import pytorch_lightning as pl
-
-    # model
-    model = nn.Sequential(
-        nn.Flatten(),
-        nn.Linear(28 * 28, 128),
-        nn.ReLU(),
-        nn.Linear(128, 10)
-    )
 
-    # data
-    dataset = datasets.MNIST('./data_folder', download=True, transform=transforms.ToTensor())
-    train, val = random_split(dataset, [55000, 5000])
+1. Download and set up your own data (:class:`~torch.utils.data.DataLoader` or `LightningModule <https://pytorch-lightning.readthedocs.io/en/stable/lightning_module.html>`_ work).
+2. Init your task.
+3. Init a :class:`flash.core.trainer.Trainer` (or a `Lightning Trainer <https://pytorch-lightning.readthedocs.io/en/stable/trainer.html>`_).
+4. Call :func:`flash.core.trainer.Trainer.fit` with your data set.
+5. Use your finetuned model for predictions
 
-    # task
-    classifier = flash.Task(model, loss_fn=nn.functional.cross_entropy, optimizer=optim.Adam)
+.. code-block:: python
 
-    # train
-    flash.Trainer().fit(classifier, DataLoader(train), DataLoader(val))
+    import torch
+    import torch.nn.functional as F
+    from flash.core.classification import ClassificationTask
+    from pytorch_lightning.metrics import Accuracy
+    from typing import Type, Callable, Mapping, Sequence, Union
+
+
+    class LinearClassifier(ClassificationTask):
+        def __init__(
+            self,
+            num_inputs,
+            num_classes,
+            loss_fn: F.cross_entropy,
+            optimizer: Type[torch.optim.Optimizer] = torch.optim.SGD,
+            metrics: Union[pl.metrics.Metric, Mapping, Sequence, None] = [Accuracy()],
+            learning_rate: float = 1e-3,
+        ):
+          super().__init__(
+            model=None,
+              loss_fn=loss_fn,
+              optimizer=optimizer,
+              metrics=metrics,
+              learning_rate=learning_rate,
+          )
+
+          self.save_hyperparameters()
+
+          self.linear = torch.nn.Linear(num_inputs, num_classes)
+
+        def forward(self, x):
+            return self.linear(x)
```

Comment:
This is part of the "create your own task", might want to remove this for now and replace once we have something good